lib/wanda/execution/server.ex

```diff
@@ -148,7 +148,7 @@ defmodule Wanda.Execution.Server do
   end
 
   defp store_and_publish_execution_result(%Result{} = result) do
```

Comment:
Should this be banged as well? `store_and_publish_execution_result!`
Or it does not matter because its private? :thinking:

Reply:
--> https://elixirforum.com/t/when-to-suffix-a-function-by-a-bang/34490